### PR TITLE
[redis] Add redis commander and port-forwarding sections for replication

### PR DESCRIPTION
**Pull request by aramalipoor**
Depends on release of https://github.com/abarcloud/abar-templates/pull/25

### CODE REVIEW

File: additional-services/redis.md [hunk 1]
```diff
@@ -39,15 +39,16 @@ $ oc port-forward redis-1-o642b 1234:6379
 Forwarding from 127.0.0.1:1234 -> 6379
 Forwarding from [::1]:1234 -> 6379
 ```
+You can also use [Redis commander template](#redis-commander) which is a web-based UI to access redis.
```
Comment (alikhajeh1) [inline additional-services/redis.md]: Redis commander template -> Redis commander
which -> that (or you can use which but with a comma, e.g.: `commander, which`)

File: additional-services/redis.md [hunk 2]
```diff
@@ -77,22 +78,49 @@ oc scale statefulset SERVICE_NAME --replicas=X
...
    SENTINEL get-master-addr-by-name mymaster
    ```
 3. Call the [`ROLE` command](https://redis.io/commands/role) on the new master to verify the role of the instance is actually a master.
 4. Retry your initial Redis command (e.g. GET, SET etc.), and if it's unsuccessful, repeat from Step 2.
+
+# Redis commander
+[Redis commander](https://github.com/joeferner/redis-commander) is web-based UI to access a Redis instance. 
+To view and manage your Redis data using Redis commander:
+
+ 1. Navigate to Add to Project > Browse Catalog > Other Services > Redis Commander
```
Comment (alikhajeh1) [inline additional-services/redis.md]: I think we should move Redis commander into the Data Stores category so it's next to the other Redis options (we do the same for PG and PG Web).

File: additional-services/redis.md [hunk 3]
```diff
@@ -77,22 +78,49 @@ oc scale statefulset SERVICE_NAME --replicas=X
 
 You need **at least 3** pods (master + slaves), and it should always be **an odd number** of replicas (5, 7, 9...).
 
-### Fail-over Procedure
+### Access from your local machine
+To do port-forwarding for a Redis replication, first find which pod is master by running command below in Terminal of any of Redis replication pods (e.g. `redis-ha-0`, etc.)  
+ 1. Make sure you have [`oc` CLI](management/cli-login.md) installed and working.
```
Comment (alikhajeh1) [inline additional-services/redis.md]: I'd delete this, it's already assumed in other parts of the docs.

File: additional-services/redis.md [hunk 4]
```diff
@@ -77,22 +78,49 @@ oc scale statefulset SERVICE_NAME --replicas=X
 
 You need **at least 3** pods (master + slaves), and it should always be **an odd number** of replicas (5, 7, 9...).
 
-### Fail-over Procedure
+### Access from your local machine
+To do port-forwarding for a Redis replication, first find which pod is master by running command below in Terminal of any of Redis replication pods (e.g. `redis-ha-0`, etc.)
```
Comment (alikhajeh1) [inline additional-services/redis.md]: Might be good to follow a consistent wording as the other section in this page that explains port-forwarding?
```If you need to connect to Redis remotely, you can use the AbarCloud CLI to start a port-forwarding session with the Redis pod's 6379 port. You can then connect to Redis as normal on localhost on the desired port:```

File: additional-services/redis.md [hunk 5]
```diff
@@ -77,22 +78,49 @@ oc scale statefulset SERVICE_NAME --replicas=X
 
 You need **at least 3** pods (master + slaves), and it should always be **an odd number** of replicas (5, 7, 9...).
 
-### Fail-over Procedure
+### Access from your local machine
+To do port-forwarding for a Redis replication, first find which pod is master by running command below in Terminal of any of Redis replication pods (e.g. `redis-ha-0`, etc.)  
+ 1. Make sure you have [`oc` CLI](management/cli-login.md) installed and working.
+ 2. Run command below to ask Sentinel about master's IP:
```
Comment (alikhajeh1) [inline additional-services/redis.md]: Run the following command to get the master's IP from Sentinel

File: additional-services/redis.md [hunk 6]
```diff
@@ -77,22 +78,49 @@ oc scale statefulset SERVICE_NAME --replicas=X
...
+    ```sh
+    oc rsh pod/redis-ha-0 redis-cli -p 26379 SENTINEL get-master-addr-by-name mymaster
+    ```
+ 3. Run command below to find master's pod name based on IP displayed in Step 2:
```
Comment (alikhajeh1) [inline additional-services/redis.md]: Run the following command to find the master's pod name based on the IP address displayed in the previous step:

File: additional-services/redis.md [hunk 7]
```diff
@@ -77,22 +78,49 @@ oc scale statefulset SERVICE_NAME --replicas=X
...
+    ```sh
+    oc get pods --show-all=false --template='{{range .items}}{{.status.podIP}}{{"\t"}}{{.metadata.name}}{{"\n"}}{{end}}'
+    ```
+ 4. Enable port forwarding using master's pod name:
```
Comment (alikhajeh1) [inline additional-services/redis.md]: Start port-forwarding using the master's pod name:

File: additional-services/redis.md [hunk 8]
```diff
@@ -77,22 +78,49 @@ oc scale statefulset SERVICE_NAME --replicas=X
...
+    ```sh
+    oc port-forward redis-ha-0 1234:6379
+    ```
+ 5. Now you have access to Redis via `127.0.0.1` on port `1234`. 
+
+You can also use [Redis commander template](#redis-commander) which is a web-based UI to access Redis replication.
```
Comment (alikhajeh1) [inline additional-services/redis.md]: Same comment as before

File: additional-services/redis.md (hunk 2 above)
Comment (aramalipoor) [reply additional-services/redis.md]: https://github.com/abarcloud/abar-templates/pull/25 Done.

**Pull request merged**